editor/src/editor/layout/assets-browser/items/material-item.tsx

```diff
@@ -54,6 +80,27 @@ export class AssetBrowserMaterialItem extends AssetsBrowserItem {
 		);
 	}
 
+	private async _computeThumbnail(): Promise<void> {

```

Comment:
Maybe you could also move the logic of this method to `cache.ts`? In case the strategy changes in the future